openshift/ftest-oc-proxy/src/test/java/org/arquillian/cube/openshift/ftest/HelloWorldTest.java

```diff
@@ -0,0 +1,49 @@
+package org.arquillian.cube.openshift.ftest;
+
+import java.util.List;
+import org.arquillian.cube.kubernetes.impl.utils.CommandExecutor;
+import org.arquillian.cube.openshift.impl.requirement.RequiresOpenshift;
+import org.arquillian.cube.requirement.ArquillianConditionalRunner;
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+@RunWith(ArquillianConditionalRunner.class)
+@RequiresOpenshift
+public class HelloWorldTest {
+
+    private CommandExecutor commandExecutor;
+
+    @Before
+    public void initCommandExecutor() {
+        commandExecutor = new CommandExecutor();
+    }
+
+    @Test
+    public void should_be_able_deploy_resources_using_oc() {
+        // when
+        final List<String> resources = commandExecutor.execCommand("oc create -f https://goo.gl/qaiGRJ");
```

Comment:
Can't we have this file referred locally? Otherwise, we would always be online to run this test.